src/JuMP.jl

```diff
@@ -517,18 +518,18 @@ Base.copy(c::SDPConstraint, new_model::Model) =
 ##########################################################################
 # ConstraintRef
 # Reference to a constraint for retrieving solution info
-immutable ConstraintRef{T<:JuMPConstraint}
-    m::Model
+immutable ConstraintRef{M<:AbstractModel,T<:JuMPConstraint}
```

Comment:
Possibly, I have some experimental prototype for a hyper-general approach to AMLs where it is, but I think it'd require some more serious re-thinking than I needed here.